src/middleware/permissionHandler.js

```diff
@@ -52,30 +52,32 @@ module.exports = function(router) {
       if (permission.type === 'read') {
         _.each(permission.resources, function(id) {
           access.submission.read_all.push(id);
+
+          // Flag this request as not having admin access through submission resource access.
+          req.submissionResourceAccessAdminBlock = req.submissionResourceAccessAdminBlock || [];
+          req.submissionResourceAccessAdminBlock.push(util.idToString(id));
         });
       }
       else if (permission.type === 'create') {
         _.each(permission.resources, function(id) {
           access.submission.create_all.push(id);
-          access.submission.read_all.push(id);
+
+          // Flag this request as not having admin access through submission resource access.
+          req.submissionResourceAccessAdminBlock = req.submissionResourceAccessAdminBlock || [];
+          req.submissionResourceAccessAdminBlock.push(util.idToString(id));
         });
       }
-      else if (permission.type === 'write') {
```

Comment:
We need to make this reverse compatible so that if anyone submits with permission type "write" it will know what to do. This seems like it breaks reverse compatability.